packages/web/src/dev/ValidatorsList.tsx

```diff
@@ -266,6 +289,26 @@ class ValidatorsList extends React.PureComponent<ValidatorsListProps & I18nProps
           <H1 style={[textStyles.center, standardStyles.sectionMarginTablet, textStyles.invert]}>
             Validator Explorer
           </H1>
+          <View>
+            <View style={[styles.links]}>
+              {networkMenu.map(([name, link]) => (
+                <View
+                  key={name}
+                  style={[
+                    styles.linkWrapper,
+                    this.props.router.pathname !== link ? styles.linkWrapperInactive : {},
+                  ]}
+                >
+                  <Button kind={BTN.DARKNAV} href={link} text={name} />
```

Comment:
btw
consider extending https://github.com/celo-org/celo-monorepo/blob/master/packages/web/src/shared/navigation.tsx to take an optional coinColor Prop. and using.

Reply:
I didn't see this component, so will be good to use it.

I think that I'm going to work on a new PR with this change and move the header outside the GraphQL query.

Reply:
sweet